src/components/assessment/AssessmentWorkspace.tsx

```diff
@@ -122,6 +122,10 @@ class AssessmentWorkspace extends React.Component<
           ? ((question as IProgrammingQuestion).answer as string)
           : (question as IProgrammingQuestion).solutionTemplate
         : null
+    const chosenMCQAnswer =
+      question.type === QuestionTypes.mcq && question.answer !== null
+        ? ((question as IMCQQuestion).answer as number)
+        : null
```

Comment:
Does this work

```typescript
const chosenMCQAnswer = (question as IMCQQuestion).answer
  ? (question as IMCQQuestion).answer
  : null
```

If it does can you also change the `const editorValue = ...` below similarly?

Don't think we need to cast it as a number. Should just wait for backend to fix.

Reply:
Actually, `(question as IMCQQuestion).answer as number || null` achieves both the typing and null checking. The same cannot be done for the editorValue as the question type must be checked. We could simplify the expression but it will be harder to read IMO.